### PR TITLE
Conscious language master slave

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well, thanks!

**Review by VladimirSlavik — APPROVED** [review]
Thank you, looks mostly good to me.

File: pyanaconda/ui/gui/spokes/advanced_storage.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def setup(self, store, disks, selected_names, protected_names):
            displayed.  This function is called every time the filter spoke
            is revisited, and thus must first do any cleanup that is necessary.
 
-           The setup function is passed a reference to the master store, a list
+           The setup function is passed a reference to the primary store, a list
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/ui/gui/spokes/advanced_storage.py:157]: This is a different context (storage) and here I think 'primary' fits better.

File: pyanaconda/modules/network/nm_client.py [hunk 2]
```diff
@@ -1315,28 +1316,28 @@ def _get_dracut_znet_argument_from_connection(connection):
...
-    :param master_specs: a list containing sepcification of master:
-                         interface name or connection uuid or both
-    :type master_specs: list(str)
-    :returns: slaves specified by name, interface and connection uuid
+    :param port_types: type of the port - NM setting "slave-type" value (eg. "team")
+    :type port_types: list(str)
+    :param controller_specs: a list containing sepcification of a controller:
+                             interface name or connection uuid or both
+    :type controller_specs: list(str)
+    :returns: portss specified by name, interface and connection uuid
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/modules/network/nm_client.py]: Fixed

**Review by jkonecny12 — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**